README.md

```diff
@@ -192,6 +192,17 @@ targets. See the documentation of the `deps` attribute for further information.
   sparingly. If this attribute is specified, then the `srcs`, `externs`, and
   `deps` attributes may not be used.
 
+### Referencing the closure library
+
+In order to use `goog.provide` and `goog.require` in your javascript code, make
+sure to add the closure library as a dependency in closure_js_library rules:
+
+```python
+deps = [
```

Comment:
Could you expand this example to include the full rule, rather than just an attribute? Otherwise people might think you're assigning a global variable.